packages/@lightningjs/ui-components/src/globals/context/index.d.ts

```diff
@@ -0,0 +1,33 @@
+type ThemeObject = any;
```

Comment:
Is this ok to be `any`, or should it at least be an `object`?

Reply:
Definitely should be object. Thanks